core/src/test/scala/com/rackspace/com/papi/components/checker/GivenAWadlWithMetaTransformsBase.scala

```diff
@@ -26,9 +26,15 @@ abstract class GivenAWadlWithMetaTransformsBase extends FlatSpec with RaxRolesBe
   val descriptionWithRaxRoles = description + " and RAX-Roles"
   val descriptionWithRaxRolesMasked = descriptionWithRaxRoles + " Masked"
 
-  val metadataWadl = <application xmlns="http://wadl.dev.java.net/2009/02" xmlns:rax="http://docs.rackspace.com/api">
+  val metadataWadl = <application xmlns="http://wadl.dev.java.net/2009/02" xmlns:rax="http://docs.rackspace.com/api"
+    xmlns:tst="http://www.rackspace.com/repose/wadl/checker/step/test">
+    <grammars>
+       <include href="src/test/resources/xsd/test-urlxsd.xsd"/>
+    </grammars>
     <resources base="https://resource.api.rackspace.com">
-      <resource id="standardResource" path="standard" rax:useMetadata="standardMeta"/>
+      <resource id="standardResource" path="standard/{id}" rax:useMetadata="standardMeta">
+            <param name="id" style="template" type="tst:UUID" required="true"/>
+      </resource>
```

Comment:
Yes, although you won't see a failure in this test since it doesn't iterate through different configuration options....which is why I wrote an different test.